src/components/try-hcp-callout/components/try-hcp-callout-compact/try-hcp-callout-compact.module.css

```diff
@@ -29,7 +29,13 @@
 
 .background {
 	composes: backgroundBase;
-	background-image: url('./img/try-hcp-callout-compact-background.svg?url');
+	background-color: var(--token-color-surface-faint);
+	background-image: linear-gradient(
+			to left,
+			var(--token-color-surface-primary),
+			transparent 80%
```

Comment:
@kellyschummer this is the gradient we will finesse for the install view 'compact' callout.

Reply:
👍 the adjustment you have here looks great!!

Reply:
Okay sweet! I'll open this for review then.